app/services/update_auction.rb

```diff
@@ -71,8 +86,12 @@ def auction_rejected?
     parsed_attributes[:status] == 'rejected'
   end
 
+  def parser
```

Comment:
Just noticed I need to be smarter about the memoization. I could fix this by memoizing `attributes` when `AuctionParser` is instantiated.